app/assets/javascripts/application.js

```diff
@@ -20,3 +20,10 @@
 //= require material-dashboard.min
 //= require perfect-scrollbar.jquery.min
 //= require_tree .
+//= require best-in
+//= require best_in_place.jquery-ui
+//= require best_in_place.purr
+
+$(document).ready(function() {
```

Comment:
dont do this here, 
create your own file, and add code there.

you can name it common.coffee